### PR TITLE
tightening the tolerance for polymer cases for testing purpose

### PR DESCRIPTION
**Pull request by GitPaean**
jenkins build this update_data please

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this update_data please

**Comment by GitPaean** [comment]
With this PR, the testing failure related to polymer cases are recovered from my local machine. @alfbr 
```
The following tests FAILED:
	 37 - compareECLFiles_flow+3D_MSW (Failed)
	 43 - compareECLFiles_flow+MSW_MODEL_1 (Failed)
	 45 - compareECLFiles_flow+FAULTS_MODEL_1 (Failed)
	 57 - compareECLFiles_flow+TEST_WSEGSICD (Failed)
```

**Comment by GitPaean** [comment]
And there is no guarantee this PR does not generate regressions on other people's local test. With tightened tolerance, it is likely to cause time step chopping. It is hard to make sure the chopping behavior are the same.

**Comment by alfbr** [comment]
Excellent! So that means that the stricter tolerances are sufficient. However, are they necessary? I will comment inline on the specific tolerances.

> And there is no guarantee this PR does not generate regressions on other people's local test.

No, but if your machine and Jenkins agree on the solution, that is a good start :-)

File: compareECLFiles.cmake [hunk 1]
```diff
@@ -263,20 +263,22 @@ add_test_compareECLFiles(CASENAME polymer_oilwater
...
+                         REL_TOL ${rel_tol}
+                         TEST_ARGS --tolerance-mb=1.e-7 )
 
 add_test_compareECLFiles(CASENAME polymer_injectivity
                          FILENAME 2D_POLYMER_INJECTIVITY
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
-                         REL_TOL ${rel_tol})
+                         REL_TOL ${rel_tol}
+                         TEST_ARGS --tolerance-mb=1.e-7 --max-strict-iter=20 --tolerance-wells=1.e-6)
```
Comment (alfbr) [inline compareECLFiles.cmake]: Do we know that all three are necessary? I.e., if you remove on of them, will the test still pass on your machine?

File: compareECLFiles.cmake [hunk 2]
```diff
@@ -263,20 +263,22 @@ add_test_compareECLFiles(CASENAME polymer_oilwater
...
 
 add_test_compareECLFiles(CASENAME polymer_simple2D
                          FILENAME 2D_THREEPHASE_POLY_HETER
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${coarse_rel_tol}
-                         TEST_ARGS --flow-newton-max-iterations=20)
+                         TEST_ARGS --tolerance-mb=1.e-7 --flow-newton-max-iterations=20)
```
Comment (alfbr) [inline compareECLFiles.cmake]: Same here, if you remove the --flow-newton-max-itrations=20, will that be sufficient for the test to pass on your machine?

Comment (GitPaean) [reply compareECLFiles.cmake]: `--flow-newton-max-iterations=20` was existing, but I can test it out.

File: compareECLFiles.cmake (hunk 1 above)
Comment (GitPaean) [reply compareECLFiles.cmake]: `--tolerance-mb=1.e-7 --max-strict-iter=20 --tolerance-wells=1.e-6` were added in the order as it shows. I can also test it by removing one of the first two.  Only the third one will not work.

**Pull request closed without merging**